lib/env.ts

```diff
@@ -96,6 +96,14 @@ const env = {
     webhook: process.env.FEATURE_TEAM_WEBHOOK === 'false' ? false : true,
     apiKey: process.env.FEATURE_TEAM_API_KEY === 'false' ? false : true,
     auditLog: process.env.FEATURE_TEAM_AUDIT_LOG === 'false' ? false : true,
+    payments:
+      process.env.FEATURE_TEAM_PAYMENTS === 'false'
+        ? false
+        : process.env.STRIPE_SECRET_KEY &&
+            process.env.STRIPE_WEBHOOK_SECRET &&
+            process.env.CRON_API_KEY
```

Comment:
`CRON_API_KEY` What is this for? Can't see it being used anywhere

Reply:
using in pages/api/webhooks/payment.ts to validate webhook calls

Reply:
removed